MM2RandoLib/Randomizers/Colors/RColors.cs

```diff
@@ -1547,22 +1547,22 @@ private void RandomizeStageColors(Patch p, Random r)
                 },
 
                 new ColorSet() {
-                    addresses = new int[] { 0xbe1f, 0x03a118, },
+                    addresses = new int[] { 0xbe1f, 0x03a118, 0x03a11b, },
```

Comment:
We've been trying to clean up random constants and refactor them to be Enums. I think we should do that here too.